### ISSUE TITLE
Show last sync time

### ISSUE THREAD

**Issue by nicolas-raoul** [aegif/CmisSync#509]

**Comment by ymolinet** [comment]
of the repo or folder/files ?

**Comment by nicolas-raoul** [comment]
Of the whole repo.
All files/folders should have the same if no error.

**Comment by ymolinet** [comment]
You have remove the lastsync from the config.xml ?

**Comment by nicolas-raoul** [comment]
Yes, it was causing a very serious bug: config.xml was deleted sometimes.
lastsync should be written to database instead.
